_posts/2016-06-20-user-testing.md

```diff
@@ -0,0 +1,149 @@
+
+At Savas Labs we've honed our approach to usability testing to balance efficiency, budget, and effectiveness. I'd like to share our process, some of my favorite resources, and some lessons we've learned along the way.
+
+## Our Process
+
+### Determine goals and user personas
+
+Focusing a usability study on a product's most critical users and functionality can help uncover the highest-impact problems without spending time and budget on lower-priority issues. To start the process, we meet with our clients and listen to them explain their purpose, describe their users, and identify potential or known issues with their website.
+
+From here, we create two very important lists:
+
+1. The client's goals for the usability study
+2. The client's users, in order of priority
+
+Let's break these down.
+
+**Goals:** These can be measurable metrics (e.g. "Increase newsletter signups" or "Increase time spent on site") or general value statements ("Provide a great experience on small screen sizes" or "Maintain a clean, modern design").
+
+**Users:** This is a list of user personas in order of priority for the particular project. A persona is more in-depth and specific than a demographic, which may represent an age group, a gender, or an occupation. A persona is defined by patterns of behavior and needs, so it may encompass multiple demographics, and one demographic may be split across several personas. The user list is prioritized using the goals of the project (unless your client is lucky enough to be able to put forward the time and money to fix all of their users' problems at once!)
+
+Once these lists are created, we review them at the beginning of every meeting (yes, every meeting!). Why? Ensuring that the entire team is completely focused on meeting the project goals at all times causes a few awesome things happen:
+
+1. Perhaps most importantly, at the end of the project everyone will be satisfied with the results, since they're founded upon pre-determined goals that were agreed upon by everyone involved.
+2. Disagreements or uncertainty can be weighed against the goals -- asking if something helps meet or doesn't help meet the goals starts a focused, productive conversation.
+3. Team members are less likely to bring up or spend time on issues that don't actually improve the end product.
+
+I should note that using user personas over demographics and repeating goals at each meeting are tips I picked up from Adam Connor's workshop ([and associated book](http://shop.oreilly.com/product/0636920033561.do)) Discussing Design. I'd highly recommend both! These practices are  an abbreviated version of what he calls a ["mini-brief"](http://www.slideshare.net/adamconnor/discussing-design-the-art-of-critique/17-The_MiniBriefA_MiniCreative_Brief_is), a one-page document detailing project goals, users, and more that is reviewed at each meeting.
```

Comment:
I think if I were just referring to the book I'd italicize it, but for the workshop I don't think quotes seem appropriate...I'll leave as is unless someone else thinks it needs to change.